modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -80,24 +67,38 @@ Supported types are:
 If you are uploading a JKS, PKCS12, or JCEKS truststore file, you must also provide the passphrase for this truststore.
 . Click *Save*.
 
-== Add a Certificate
+[[add_keystore]]
+== Add a Keystore
 
-Add a PEM or JKS certificate to the secret group. 
+Specify the type of keystore to add to the secret group. The keystore is the combination of the authorization certificate, its corresponding private keys, and the certification authority's path.
 
-. In the *Secret Groups* list view, select the secret group to which to add a certificate. 
-. Select *Certificate* in the menu on the left, and click *Add Certificate*. 
-. In the *Add Certificate* screen, add the required information. 
+. In the *Secret Groups* list view, select the secret group to which to add a keystore. 
+. Select *Keystore* in the menu on the left, and click *Add Keystore*. 
+. In the *Name* field, enter a name for your keystore.
+. In the *Type* field, select the keystore type from the drop-down menu.
 +
-* *Name* - Enter a name for your certificate.
-* *Type* -  Select the certificate type from the drop-down menu.
+Supported types include the following:
```

Comment:
you don't need "the following" (fine to keep for consistency with other sections or topics).